.github/workflows/release.yaml

```diff
@@ -14,6 +11,8 @@ jobs:
     steps:
       - name: Checkout Repo
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
```

Comment:
@lucacavallaro should we backport this?

Reply:
we already talked about it, the workflow only works if I set `fetch-depth: 0` in the checkout